src/media/js/site/components/header.js

```diff
@@ -39,24 +40,59 @@ class HeaderLogin extends React.Component {
     loginHandler: React.PropTypes.func.isRequired,
     logoutHandler: React.PropTypes.func.isRequired,
     isLoggedIn: React.PropTypes.bool.isRequired,
+    showingUserDropdown: React.PropTypes.bool.isRequired,
+    toggleUserDropdown: React.PropTypes.func.isRequired,
   };
+
+  renderAnonNav() {
+    return (
+      <nav className="header--nav">
+        <ul>
+          <li><LoginButton isSignup={true} {...this.props}/></li>
+          <li><LoginButton {...this.props}/></li>
+        </ul>
+      </nav>
+    );
+  }
+
+  handleLogoutClick = evt => {
+    this.props.toggleUserDropdown();
+    this.props.logoutHandler(evt);
+  }
+
+  renderUserNav() {
+    let dropdownClasses = classNames({
+      'header--nav': true,
+      'header--nav--show-user-dropdown': this.props.showingUserDropdown
+    });
+
+    return (
+      <nav className={dropdownClasses}>
```

Comment:
1. i don't think this is as important as everything is in the top right which is consistent enough. the reason fireplace has a user dropdown because there are unique user related stuff that doesn't belong in the main nav. here the only useful thing to show is the Log Out button

2. we can probably mitigate that by reducing the wordmark size, or stacking the username + logout button.

i don't think it adds too much utility for this project, but it adds more code + nav fluff